addons/web/models/models.py

```diff
@@ -182,7 +182,7 @@ def read_progress_bar(self, domain, group_by, progress_bar):
                 group_by_value = group_by_value[1] # FIXME should use technical value (0)
 
             if field_type == 'many2many' and isinstance(group_by_value, list):
-                group_by_value = tuple(group_by_value) or False
+                group_by_value = str(tuple(group_by_value) or False)
```

Comment:
only the `tuple()` needs to be stringified, `False` can be left as-is, I also suggest adding a comment stating that the tuple is stringified because the result of this function is sent over jsonrpc to the web client and JSON does not support tuples as dict keys

Reply:
@Elkasitu  👍